packages/web-pkg/src/components/Spaces/QuotaModal.vue

```diff
@@ -151,7 +151,11 @@ export default defineComponent({
           })
         }
         spacesStore.updateSpaceField({ id: space.id, field: 'spaceQuota', value: driveData.quota })
-        updateResourceField({ id: space.id, field: 'spaceQuota', value: driveData.quota })
+        updateResourceField<SpaceResource, any>({
```

Comment:
Do you really need to specify `any` here? Isn't partial specialization possible?

Reply:
I tried, but couldn't get the types to work. Maybe the types of `updateResourceField` are not quite correct? I'm a bit stuck here.

Reply:
As discussed, found a solution: https://github.com/owncloud/web/pull/10715/commits/b3f89fa5cb1913a9e345d771edc63bd93798f243